robottelo/test.py

```diff
@@ -170,7 +171,10 @@ def setUpClass(cls):  # noqa
     def setUp(self):  # noqa
         """We do want a new browser instance for every test."""
         if self.driver_name.lower() == 'firefox':
-            self.browser = webdriver.Firefox()
+            self.browser = webdriver.Firefox(
+                firefox_binary=webdriver.firefox.firefox_binary.FirefoxBinary(
+                    self.driver_binary
```

Comment:
No, it wont fail. FirefoxBinary() init is robust enough to handle None. There is logic within for this. By omitting webdriver_binary you stay on the default